pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -310,6 +310,22 @@ func toBeDisabled() toolchainv1alpha1.Condition {
 	}
 }
 
+func toBeProvisionedNotificationCreated() toolchainv1alpha1.Condition {
+	return toolchainv1alpha1.Condition{
+		Type:   toolchainv1alpha1.MasterUserRecordUserProvisionedNotificationCreated,
+		Status: corev1.ConditionTrue,
+		Reason: toolchainv1alpha1.MasterUserRecordNotificationCRCreatedReason,
+	}
+}
+
+func toBeProvisionedNotificationFailed() toolchainv1alpha1.Condition {
+	return toolchainv1alpha1.Condition{
+		Type:   toolchainv1alpha1.MasterUserRecordUserProvisionedNotificationCreated,
+		Status: corev1.ConditionFalse,
+		Reason: toolchainv1alpha1.MasterUserRecordNotificationCRCreationFailedReason,
```

Comment:
Add a message too. When there is an error then we set the error message as the condition message.

Reply:
Good idea, added it here: 84fadd766bb49d3a0303a5d271948dbe8e461a5c